ghost/core/test/unit/server/services/stats/posts.test.js

```
@@ -959,53 +959,148 @@ describe('PostsStatsService', function () {
+                timezone: 'UTC',
+                limit: 5
+            });
+
+            // Basic verification that the method works for cross-post scenarios
+            assert.ok(Array.isArray(result), 'Result should be an array');
+            
+            // This test verifies cross-post attribution handling
+            // post1: should get credit for free signups
+            // post2: should get credit for paid conversions
+            if (result.length > 0) {
+                assert.ok(result[0].hasOwnProperty('members'), 'Results should have members property');
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)